src/main/java/com/google/sps/servlets/ProfileServlet.java

```diff
@@ -91,7 +91,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String errorMessage = "User needs to log in to see profile.";
       responseMap.put("error", errorMessage);
```

Comment:
We should discuss error messages/logging so that we are consistent

Reply:
That's a good idea.